### PR TITLE
add description about scene context

### PR DESCRIPTION
**Pull request by Licini**
As tilted

### CODE REVIEW

File: docs/userguide/basics.visualisation.rst [hunk 1]
```diff
@@ -114,3 +114,31 @@ To use a different scene object as the parent, the parent attribute of the scene
 >>> boxobj = scene.add(box, parent=pointobj)
 >>> boxobj.parent
 PointObject
+
+
+Scene Context
+=============
+
+Depends on where the code is executed, the "scene" will detect the current visualisation context.
```
Comment (tomvanmele) [inline docs/userguide/basics.visualisation.rst]: Depending

File: docs/userguide/basics.visualisation.rst [hunk 2]
```diff
@@ -114,3 +114,31 @@ To use a different scene object as the parent, the parent attribute of the scene
...
+If the code is executed in the Rhino, the scene context will be automatically set as "Rhino". 
+
+>>> from compas.scene import Scene
+>>> from compas.geometry import Box
+>>> scene = Scene()
+>>> scene.context
+Rhino
+
+Depends on the context, the different implementations of the scene objects will be used.
```
Comment (tomvanmele) [inline docs/userguide/basics.visualisation.rst]: For every context, the appropriate scene object implementations will be used automatically.

File: docs/userguide/basics.visualisation.rst [hunk 3]
```diff
@@ -114,3 +114,31 @@ To use a different scene object as the parent, the parent attribute of the scene
...
+
+>>> box = Box.from_width_height_depth(1, 1, 1)
+>>> scene.add(box)
+<compas_rhino.scene.BoxObject>
+
+User can also mannually set the scene context by using the following code:
```
Comment (tomvanmele) [inline docs/userguide/basics.visualisation.rst]: Users can also set the scene context manually:

**Review by tomvanmele — APPROVED** [review]

**Comment by Licini** [comment]
@tomvanmele all done and merging.

**Pull request merged**